src/ssl.c

```diff
@@ -18718,6 +18718,8 @@ size_t wolfSSL_get_client_random(const WOLFSSL* ssl, unsigned char* out,
         ssl->keys.encryptionOn = 0;
         XMEMSET(&ssl->msgsReceived, 0, sizeof(ssl->msgsReceived));
 
+        InitSSL_Suites(ssl);
```

Comment:
Does return value of InitSSL_Suites need to be checked?

Reply:
fixed